hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -261,6 +265,7 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 		hyperv1.NodePoolValidGeneratedPayloadConditionType: r.validGeneratedPayloadCondition,
 		hyperv1.NodePoolReachedIgnitionEndpoint:            r.reachedIgnitionEndpointCondition,
 		hyperv1.NodePoolAllMachinesReadyConditionType:      r.machineAndNodeConditions,
+		hyperv1.NodePoolValidPlatformConfigConditionType:   r.validPlatformConfigCondition,
```

Comment:
is there a plan to consolidate this with setPlatformConditions somehow?

Reply:
setPlatformConditions sets different conditions for each platform, right? I wanted this to be the same single condition for all platforms.